gems/rails/CVE-2008-4094.yml

```diff
@@ -0,0 +1,29 @@
+---
+gem: rails
+cve: 2008-4094
+ghsa: xf96-32q2-9rw2
+url: http://www.rorsecurity.info/2008/09/08/sql-injection-issue-in-limit-and-offset-parameter
```

Comment:
The patches referenced in the [lighthouse](https://rails.lighthouseapp.com/projects/8994/tickets/288) link reference `activerecord`. Should probably be moved to `gems/activerecord/`.

Reply:
Made the changes.